### PR TITLE
Add FireShare

### PR DESCRIPTION
**Pull request by mzch**
Add `BlackUpload` and `FireShare`

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the following requirements (`Demo` and `Clients` are optional, don't add a duplicate `Source code` link if it is the same as the main link):

``- [Name](http://homepage/) - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

- [x] If your additions depend on proprietary services outside the user's control, they must be marked `⚠`.

``- [Name](http://homepage/) `⚠` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and marked `⊘ Proprietary`:

``- [Name](http://homepage/) `⊘ Proprietary` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``

- [x] Your additions are ordered alphabetically.
- [x] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [awesome-analytics](https://github.com/onurakpolat/awesome-analytics) (analytics), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [x] The `Language` tag is the main server-side requirement for the software - don't include frameworks/specifc dialects.
- [x] Any licenses you have added are in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items. If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is "Add aaa to bbb" for adding software aaa to section bbb. Do not call it "Update README.md".

### CODE REVIEW

**Comment by mzch** [comment]
`BlackUpload` has no LICENSE file but `License` section in README.md is pointed as `MIT`.

**Comment by slel** [comment]
Maybe fix "anonymus" -> "anonymous" and decrease capitalisation?
```diff
- An Anonymus File Uploader Script with Multi-File Upload Support
+ An anonymous file uploader script with multi-file upload support
```
```diff
- A Full-Stack, Pub-Sub, Real-time secure file sharing system.
+ A full-stack, pub-sub, real-time secure file sharing system.
```

**Comment by mzch** [comment]
> Maybe fix "anonymus" -> "anonymous" and decrease capitalisation?

This is typo on the repository. Fixed

> ```diff
> - An Anonymus File Uploader Script with Multi-File Upload Support
> + An anonymous file uploader script with multi-file upload support
> ```
> 
> ```diff
> - A Full-Stack, Pub-Sub, Real-time secure file sharing system.
> + A full-stack, pub-sub, real-time secure file sharing system.
> ```

Mm... These are quoted from the repositories `About` section.

**Comment by slel** [comment]
In their own "About", their short description is a title,
so it makes sense for them to use title case for it.

Here, each project's short description becomes part of a list,
and it hurts our eyes if each list item wants to be a title.

But maybe that's just my view; I'll let others chime in.

I'll separately submit a change to the pull request template
to have it recommend all new submissions use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
for the project's short description.

**Comment by mzch** [comment]
OK. fixed.

**Comment by slel** [comment]
Thanks!  See #2341 for making this a general recommendation.

File: README.md [hunk 1]
```diff
@@ -744,10 +744,12 @@ _See https://github.com/n1trux/awesome-sysadmin#distributed-filesystems_
 
 #### Single-click/drag-n-drop upload
 
+- [BlackUpload](https://github.com/FarisCode511/BlackUpload) - An anonymous file uploader script with multi-file upload support. `MIT` `PHP`
```
Comment (nodiscc) [inline README.md:747]: https://github.com/FarisCode511/BlackUpload was archived by the owner

File: README.md [hunk 2]
```diff
@@ -744,10 +744,12 @@ _See https://github.com/n1trux/awesome-sysadmin#distributed-filesystems_
...
 - [BoZoN](https://github.com/broncowdd/BoZoN) - Minimalist Drag and drop file sharing app. `AGPL-3.0` `PHP`
 - [Coquelicot](https://coquelicot.potager.org/) - Coquelicot is a “one-click” file sharing web application with a focus on protecting users’ privacy. ([Source Code](https://coquelicot.potager.org/gitweb/?p=coquelicot.git)) `AGPL-3.0` `Ruby`
 - [elixire](https://elixi.re) - Simple yet advanced screenshot uploading and link shortening service. ([Source Code](https://gitlab.com/elixire/elixire), [Clients](https://gitlab.com/elixire/elixiremanager)) `AGPL-3.0` `Python`
 - [fibridge](https://github.com/anderspitman/fibridge-proxy-rs) - Stream huge files out of your browser without having to upload. `MIT` `Rust`
+- [FireShare](https://auspham.dev/FireShare/) - A full-stack, pub-sub, real-time secure file sharing system. Using NodeJS, ReactJS, SocketIO, MongoDB. Chai, WebdriverIO for testing. ([Source Code](https://github.com/rockmanvnx6/FireShare)) `Nodejs` `MIT`
```
Comment (nodiscc) [inline README.md:752]: Please remove the technical stack description `Using NodeJS, ReactJS, SocketIO, MongoDB. Chai, WebdriverIO for testing.`

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mzch) [reply README.md]: deleted. ;)

File: README.md (hunk 1 above)
Comment (mzch) [reply README.md]: deleted.

**Review by nodiscc — APPROVED** [review]

**Pull request merged**